spec/Section 6 -- Execution.md

```diff
@@ -769,13 +772,13 @@ MergeSelectionSets(fields):
 
 ### Handling Field Errors
 
-["Field errors"](#sec-Errors.Field-errors) are raised from a particular field
-during value resolution or coercion. While these errors should be reported in
-the response, they are "handled" by producing a partial response.
+A _field error_ is an error raised from a particular field during value
+resolution or coercion. While these errors should be reported in the response,
+they are "handled" by producing a partial response.
 
-Note: This is distinct from ["request errors"](#sec-Errors.Request-errors) which
-are raised before execution begins. If a request error is encountered, execution
-does not begin and no data is returned in the response.
+Note: This is distinct from a _request error_ which is raised before execution
+begins. If a request error is encountered, execution does not begin and no data
+is returned in the response.
```

Comment:
```suggestion
Note: This is distinct from a _request error_ which is raised before execution
begins. If a _request error_ is encountered, execution does not begin and no data
is returned in the response.
```

I'm indifferent to this suggestion since it's already linked in the paragraph, feel free to dismiss

Reply:
I'll dismiss. My rule of thumb is to link every normative reference (i.e. in an algo) and to link the first or most prominent reference in a particular prose